### PR TITLE
DOC-2239: Remove deprecated remove_trailing_brs option.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2239:

Changes:
* remove documentation `remove_trailing_brs.adoc` to support TINY-10454 as this option was marked as deprecated in a previous release- now this  `function` has now been removed from the `code` for `TinyMCE 7.0`.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/partials/configuration/remove_trailing_brs.adoc [hunk 1]
```diff
@@ -1,6 +1,9 @@
 [[remove_trailing_brs]]
 == `+remove_trailing_brs+`
 
+[IMPORTANT]
+This option has been deprecated in {productname} 7.0.
```
Comment (lorenzo-pomili) [inline modules/ROOT/partials/configuration/remove_trailing_brs.adoc:5]: `remove_trailing_brs` was already deprecated with `TINY-10454` this function has been removed

Comment (kemister85) [reply modules/ROOT/partials/configuration/remove_trailing_brs.adoc:5]: @lorenzo-pomili yes, but the functions fix-version is set to 7.0? And we have not documented the removal of this option. So we still need to add it to the release-notes for 7.0, and insert it to the `migration-guide` for v6 to v7 users as a deprecated option.

Comment (FarzadHayat) [reply modules/ROOT/partials/configuration/remove_trailing_brs.adoc:5]: If the option has been removed, then perhaps `modules/ROOT/partials/configuration/remove_trailing_brs.adoc` file could be deleted altogether in TinyMCE 7.0 Docs.

**Review by FarzadHayat — APPROVED** [review]

File: modules/ROOT/partials/configuration/remove_trailing_brs.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/partials/configuration/remove_trailing_brs.adoc:5]: @lorenzo-pomili good pickup, I'll update accordingly, sorry for the confusion.

Comment (kemister85) [reply modules/ROOT/partials/configuration/remove_trailing_brs.adoc]: @lorenzo-pomili As discussed, ive `deleted` all reference to `remove_trailing_brs` from the `7.0` docs project, and have added a checkbox for use to ensure this is noted in the `migration guide`.

**Review by lorenzo-pomili — APPROVED** [review]

**Review by ShiridiGandham — COMMENTED** [review]
LGTM

**Pull request merged**